scripts/UpdateRepos.ps1

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env powershell
+
+<#
+.SYNOPSIS
+    Updates each repo Universe builds to new dependencies.props
+.PARAMETER Source
+    The NuGet package source to find the lineup on.
+.PARAMETER LineupID
+    The ID of the Lineup to determine which versions to use.
+.PARAMETER LineupVersion
+    The version of the Lineup to be used.
+#>
+[cmdletbinding(SupportsShouldProcess = $true)]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$Source,
+    [Parameter(Mandatory=$true)]
+    [string]$LineupID,
+    [Parameter(Mandatory=$true)]
+    [string]$LineupVersion,
+    [string[]]$GitCommitArgs = @()
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2
+
+Import-Module "$PSScriptRoot/common.psm1" -Scope Local -Force
+
+$RepoRoot = Resolve-Path "$PSScriptRoot\.."
+$ModuleDirectory = Join-Path $RepoRoot "modules"
+
+Push-Location $ModuleDirectory
+try {
+    # Init all submodules
+    Invoke-Block { & git submodule update --init }
+
+    $update_errors = @()
+    $submodules = Get-Submodules $ModuleDirectory
+    foreach($submodule in $submodules)
+    {
+        Push-Location $submodule.path
+        try {
+            $depsFile = Join-Path (Join-Path $($submodule.path) "build") "dependencies.props"
```

Comment:
We should skip the update if this file doesn't exist

Reply:
Addressed.